internal/handlers/pg/cmd_query.go

```diff
@@ -47,8 +52,19 @@ func (h *Handler) CmdQuery(ctx context.Context, query *wire.OpQuery) (*wire.OpRe
 					"readOnly", false,
 					"ok", float64(1),
 				))},
-			}
-			return reply, nil
+			}, nil
+		case "saslStart":
+			var emptyPayload types.Binary
+
+			return &wire.OpReply{
```

Comment:
But that completely ignores authentication, no? Shouldn't we call `common.SASLStartPlain` or something?

Reply:
Please see https://github.com/FerretDB/FerretDB/blob/main/internal/handlers/pg/msg_saslstart.go